src/config/crd/bases/goharbor.goharbor.io_inspectionpolicies.yaml

```diff
@@ -298,6 +298,9 @@ spec:
                   riskImage:
                     description: Image of the risk.
                     type: string
+                  workloadScanner:
```

Comment:
I don't see `inspectionpolicy_types.go `changes here, CRD bases have to be generated from types.

https://github.com/vmware-tanzu/cloud-native-security-inspector/blob/main/src/api/v1alpha1/inspectionpolicy_types.go